### ISSUE TITLE
salt doc err

### ISSUE THREAD

**Issue by fmnisme** [saltstack/salt#33085]
https://docs.saltstack.com/en/latest/ref/netapi/all/salt.netapi.rest_cherrypy.html#salt.netapi.rest_cherrypy.app.Keys

```
    Mock out specified imports

    This allows autodoc to do its thing without having oodles of req'd installed libs. This doesn't work with import * imports.

    http://read-the-docs.readthedocs.org/en/latest/faq.html#i-get-import-errors-on-libraries-that-depend-on-c-modules

```

**Comment by jfindlay** [comment]
@fmnisme, thanks for reporting.

**Comment by cachedout** [comment]
@whiteinge Do you know what's going on here by chance? Could this be related to the decorator on that method?

**Comment by whiteinge** [comment]
You're right, it's the decorator. We mock the CherryPy import so when Sphinx tries to auto-document that class it hits the mocked decorator instead of the code. 💭

**Comment by cachedout** [comment]
This might get solved by #33132.
